core/bin/external_node/src/config/mod.rs

```diff
@@ -375,11 +375,35 @@ impl RequiredENConfig {
     }
 }
 
+/// Configuration for Postgres database.
+/// While also mandatory, it historically used different naming scheme for corresponding
+/// environment variables.
+/// Thus it is kept separately for backward compatibility and ease of deserialization.
+#[derive(Debug, Clone, Deserialize, PartialEq)]
+pub struct PostgresConfig {
+    pub database_url: String,
+    pub max_connections: u32,
+}
+
+impl PostgresConfig {
+    pub fn from_env() -> anyhow::Result<Self> {
+        Ok(Self {
+            database_url: env::var("DATABASE_URL")
```

Comment:
But here you use convention from the main database

Reply:
<img width="807" alt="image" src="https://github.com/matter-labs/zksync-era/assets/12111581/cdb6f5e8-3ce1-4515-ad49-1ca1fed24d4b">

Reply:
Yeah, but our partners use `EN_DATABASE`

Reply:
Could you please show where this variable is actually used?
Before this PR DAL itself has been loading the variable without any prefixes, and our [config samples](https://github.com/matter-labs/zksync-era/blob/main/docs/external-node/prepared_configs/mainnet-config.env) in docs also do not have prefixes for the database.

This was done this way _exactly_ to be backward compatible.